hologram_dashboard/fetcher/flags.h

```diff
@@ -0,0 +1,22 @@
+/*
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*/
+
+#pragma once
+#include <gflags/gflags.h>
+
+DECLARE_string(chipper_batch_job_cell);
+DECLARE_string(chipper_gdpr_batch_job_cell);
+DECLARE_string(config_file_path);
```

Comment:
I found ABSL_FLAG is public lib.

Can you switch to ABSL_FLAG? Its usage is very similar to this one.

https://abseil.io/docs/cpp/guides/flags

Reply:
is there a reason why we would want to switch to absl_flag over gflags?

Reply:
gflags are used ~2 years ago. Now all Google codes are switching to use ABSL_FLAG. We should keep in sync.

Originally I thought ABSL_FLAG is not a public lib. My bad.